src/trade.js

```diff
@@ -152,7 +161,8 @@ class Trade {
     var reservation = quote.delegate.reserveReceiveAddress();
 
     var processTrade = function (res) {
-      var trade = new quote._TradeClass(res, quote.api, quote.delegate);
+      var trade = new quote._TradeClass(null, quote.api, quote.delegate);
+      trade.setFromAPI(res);
```

Comment:
`setFromAPI()` in bitcoin-exchange-client doesn't do anything useful except toggling the `_fromAPI` flag for bookkeeping purposes. It's the job of each subclass to implement it (which I still need to do for Coinify & SFOX).